Floofbot/Modules/Fun.cs

```diff
@@ -168,5 +168,172 @@ public async Task RequestCatFact()
             }
             await Context.Channel.SendMessageAsync(fact);
         }
+
+        [Command("minesweeper")]
+        [Summary("Minesweeper minigame")]
+        public async Task Minesweeper([Summary("grid x")]int gridx, [Summary("grid y")]int gridy, [Summary("bomb count")]int bombs)
```

Comment:
What happens if it's not an int? Is it just logged as an error and the bot acts like nothing happened?
I've been assuming the args had to be strings for some reason.

Reply:
The bot logs an error and it will not respond to the command